src/lib/blink.ts

```diff
@@ -1,5 +1,6 @@
-import blinking = require('blinking');
+import blinking = require('blinking'); // blinking is a 'coffee' module and can only be imported this way
```

Comment:
This [blinking module](https://github.com/balena-io-modules/blinking) is actually a bit of a problem. It's an Javascript module with no typings first of all, and we also don't have type declaration file for it. It also hasn't been updated in 5 years. This [TODO comment](https://github.com/balena-os/balena-supervisor/blob/master/test/07-blink.spec.ts#L16) from two years ago has remained all this time. In my test refactor branch that I've been looking at, the blink test straight up doesn't pass TypeScript checks :|

Nothing wrong with your change here, but you were mentioning figuring out which external modules aren't TypeScript-safe and seeing if there were any alternatives, and this module is one I've experienced friction with in the past.

Reply:
It hasn't been updated in so long because it hasn't needed to be :P  It's our own module as well (it was part of the supervisor before being separated out) so it's easy enough to convert to js and/or typescript and have it generate types from one of those

Reply:
https://github.com/balena-io-modules/blinking/pull/5

Reply:
Nice one @Page-

For context, the only reason I left this comment is to leave a reminder to myself to remove the dependency or update the module to typescript. I will probably edit and make the same with other non-typescript dependencies

Reply:
@Page- Thanks for the speedy update Page!